src/services/fs/mod.rs

```diff
@@ -92,3 +92,4 @@ pub use backend::Builder;
 
 mod dir_stream;
 mod error;
+pub mod nfs;
```

Comment:
```suggestion
mod nfs;
```

We don't want user to access this mod.